web/src/RootSwitcher.tsx

```diff
@@ -84,11 +84,8 @@ const RootSwitcher = ({ setContentLanguage }: RootSwitcherProps): ReactElement =
 
         {/* Redirects */}
         <Route path='/' element={<Navigate to={fixedCityPath ?? landingPath} replace />} />
-        <Route path={LANDING_ROUTE} element={<Navigate to={fixedCityPath ?? landingPath} replace />} />
+        <Route path={RoutePatterns[LANDING_ROUTE]} element={<Navigate to={fixedCityPath ?? landingPath} replace />} />
         <Route path='/:cityCode' element={<Navigate to={fixedCityPath ?? language} replace />} />
```

Comment:
This now also handles redirecting `/landing`. Perhaps we should make that clear using a comment :see_no_evil: